src/client/components/EventCardText/EventCardText.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './EventCardText.styles.css';
+
+export const EventCardText = (props) => {
+  return (
+    <div>
+      <div className="event-card-title">{props.title}</div>
+      <div className="event-card-date-time">{props.datetime}</div>
+      <div className="event-card-mentor">{props.mentor}</div>
```

Comment:
It would be best if you only pass mentor name here and not the whole string. Example: you pass in `Mentor name` and the `EventCardText` component actually adds `Mentor:` before it without it being passed through the text.